pkg/function/delete_data.go

```diff
@@ -67,6 +68,10 @@ func (*deleteDataFunc) Name() string {
 }
 
 func deleteData(ctx context.Context, cli kubernetes.Interface, tp param.TemplateParams, reclaimSpace bool, namespace, encryptionKey string, targetPaths, deleteTags, deleteIdentifiers []string, jobPrefix string, podOverride crv1alpha1.JSONMap) (map[string]interface{}, error) {
```

Comment:
```suggestion
func deleteData(
	ctx context.Context,
	cli kubernetes.Interface,
	tp param.TemplateParams,
	reclaimSpace bool,
	namespace,
	encryptionKey string,
	targetPaths,
	deleteTags,
	deleteIdentifiers []string,
	jobPrefix string,
	podOverride crv1alpha1.JSONMap,
) (map[string]interface{}, error) {
```
No changes - just formatting